packages/manager/src/features/CloudPulse/shared/CloudPulseDashboardSelect.tsx

```diff
@@ -10,16 +10,24 @@ import type { Dashboard, FilterValue } from '@linode/api-v4';
 
 export interface CloudPulseDashboardSelectProps {
   defaultValue?: Partial<FilterValue>;
-  handleDashboardChange: (
+  disabled?: boolean;
+  handleDashboardChange?: (
     dashboard: Dashboard | undefined,
     savePref?: boolean
   ) => void;
   savePreferences?: boolean;
+  serviceIntegration?: boolean;
```

Comment:
I think we can just control everything based on serviceIntegration Prop and no need of disabled prop itself